python/mleap/sklearn/preprocessing/data.py

```diff
@@ -1102,8 +1102,8 @@ def fit_transform(self, X, y=None, **fit_params):
     def serialize_to_bundle(self, path, model_name):
         # compile tuples of model attributes to serialize
         attributes = list()
-        attributes.append(("labels", self.labels.keys()))
-        attributes.append(("values", Vector(self.labels.values())))
+        attributes.append(("labels", list(self.labels.keys())))
+        attributes.append(("values", Vector(list(self.labels.values()))))
```

Comment:
do we need both vector and list or just one will be fine? "labels" and "values" should be similar in how they are serialized i think.

Reply:
In python 3 OrderedDict.values() is returning an iterator and that's not serializable to json. So I need to materialize the iterator into a list by calling `list()` on it. 

Is there a reason why we're using a Vector ? I see from https://github.com/combust/mleap/blob/5e764fbaa14942389e24d28570cde750ad22f667/python/mleap/bundle/serialize.py#L66 and https://github.com/combust/mleap/blob/5e764fbaa14942389e24d28570cde750ad22f667/python/mleap/bundle/serialize.py#L83 that Vector and list are serialized differently. 
On the other end, I noticed that tests pass in both cases:
1. `        attributes.append(("values", Vector(list(self.labels.values()))))`
2. `        attributes.append(("values", list(self.labels.values())))` 

if Vector is unnecessary I can move to a simple `list` as in 2)
Otherwise i'm fine having both Vector (for serialization) and `list` for iterator to list conversion as in case 1)

Reply:
Ah okay, sorry, I see the difference in serialize.py. We can leave it as is for now.